docs/development/getting_started.rst

```diff
@@ -199,7 +199,7 @@ across the full stack.
 
 If you want to receive OSSEC alerts or change any other settings, you will need
 to fill out your local copy of
-``securedrop/install_files/ansible_base/staging-specific.yml``.
+``./install_files/ansible_base/staging-specific.yml``.
```

Comment:
It's arguable that including the single period for current working directory - `./install_files` vs. the absolute sufficiency of `install_files/` is complicating with two extra characters and shell conventions. Just pointing out there's another choice ;)